.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode8
+osx_image: xcode9.2
```

Comment:
Since we plan on supporting Xcode 9.0, this should use `xcode9` (or `xcode9.0`, whichever Travis requires).

You'll also need to update the README to explicitly change our minimum requirements to Xcode 9. Though I'm curious: given that there were no backwards-incompatible code changes, is there a chance this project would still build on Xcode 8?

Reply:
I am curious, why are you so curious? (Sry could not resist, hope you know the movie.)

Yeah, you are absolutely having a point there. It runs on Xcode 8 as well. I was triggered to change that because of the hint Travis CI gives you:

>  This job was configured to run on an OS X image that was retired on November 28, 2017. It was routed to our Xcode 8.3 image.

So I thought, as we are on Xcode 9.2 right now, this should not cause too much damage. But sticking to Xcode 9 sounds feasible to me.